warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -452,17 +454,14 @@ models:
           Rows with `true` in this column have a duplicate primary key in dim_trips;
           i.e., `trip_id` is duplicated within an individual feed instance.
           Treat these rows with caution.
-      - name: n_stops
-        description: |
-          A distinct count of stop_id by trip
-          Because this represents distinct stop_id values, it is not appropriate to sum across different trips.
-      - name: n_stop_times
-        description: |
-          A count of stop events by trip
+      - name: num_distinct_stops_served
+        description: '{{ doc("column_num_distinct_stops_served") }}'
+      - name: num_stop_times
+        description: '{{ doc("column_num_stop_times") }}'
       - name: trip_first_departure_sec
         description: |
-          The number of seconds after midnight on `service_day` in `feed_timezone` at
-          which this trip's first stop departure occurred.
+          The number of seconds after 12 hours before noon (usually midnight) on `service_date`
+          in `feed_timezone` at which this trip's first stop departure occurred.
```

Comment:
nit: this is hard to understand; is it `(noon - 12 hrs) + trip_first_departure_sec seconds`? Maybe you could include that (like the flex docs)

Reply:
Actually this is pretty standard verbiage in the `dims` yaml, would rather maintain consistency for now